### ISSUE TITLE
Focus order in the Dialogs

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#219]
@jobara said

> Although I find the tab order is a bit unexpected with the "close" button being the last item in the tab order.

I just assumed that focusable content in the dialog should come before the close button. Otherwise there is no strong reason for it to be last.

@jobara @jhung @terracoda should focus go to the close button first?

**Comment by terracoda** [comment]
@jessegreenberg, focus typically goes on the first focusable item according to the [Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal). There may be a reason not to do that, for example, when the focusable item falls below the fold (out of view) in a long dialog that scrolls.

Whatever gets keyboard focus should be visible.

**Comment by jessegreenberg** [comment]
@terracoda that sounds good.  Right now the links come first, so they get focus first, but the button could come first instead.  Do you have a preference for Dialogs?

**Comment by terracoda** [comment]
@jessegreenberg, I think placing the focus on the links is fine. 
Where is the close button in the actual content order?

**Comment by jessegreenberg** [comment]
Ok, sounds good.  The close button is the very last thing in the dialog content order.

**Comment by terracoda** [comment]
@jessegreenberg, I think that's perfect.

**Comment by jessegreenberg** [comment]
Ok, thanks @terracoda - closing.
